warehouse/core/src/test/java/datawave/util/CompositeTimestampTest.java

```diff
@@ -0,0 +1,27 @@
+package datawave.util;
+
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.Test;
+import org.locationtech.jts.util.Assert;
+
+public class CompositeTimestampTest {
```

Comment:
Lets create some tests that test either side of the bounds for the event data and the ageoff date